recipes/pyflowy/recipe.yaml

```diff
@@ -0,0 +1,88 @@
+schema_version: 1
+
+context:
+  name: pyflowy
+  version: 0.0.1
+  build_num: 0
+  shasum: babe8f58e1d2cc3784a87a50bfbb904cac088ad51259a7a2ccb835d5e3f0dfd1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/pyflowy-${{ version }}.tar.gz
+  sha256: ${{ shasum }}
+
+build:
+  skip:
+    - if: python < 3.9
```

Comment:
Technically you don't need to have the `python < 3.9` check since conda-forge only builds packages for python >=3.9